File: voice/outbound/ruby/outbound.rb
require "net/http"
require "uri"

uri = URI.parse("https://rest.nexmo.com/call/json")
params = {
    "api_key" => API_KEY,
    "api_secret" => API_SECRET,
    "to" => YOUR_NUMBER,
    "from" => NEXMO_NUMBER,
    "answer_url" => "http://example.com/outbound.vxml"
}

response = Net::HTTP.post_form(uri, params)

puts response.body

File: voice/outbound/python/outbound.py
import urllib

params = {
    'api_key': API_KEY,
    'api_secret': API_SECRET,
    'to': YOUR_NUMBER,
    'from': NEXMO_NUMBER,
    'answer_url': 'http://example.com/outbound.vxml'
}

url = 'https://rest.nexmo.com/call/json?' + urllib.urlencode(params)

response = urllib.urlopen(url)
print response.read()
